modules/core/shared/src/main/scala/coursier/maven/Pom.scala

```diff
@@ -216,30 +218,17 @@ object Pom {
 
       profiles <- xmlProfiles.eitherTraverse(profile)
 
-      extraAttrs <- properties
```

Comment:
`Pom` and `PomParser` do not parse the extra-attributes any more. But `SbtPom` and `SbtPomParser` do.

Reply:
I pushed a few commits that slightly change that: extra attributes are handled by `SbtMavenRepository`, that looks at `Project#properties`, and reads the extra attributes from there.